src/client/src/App/LiveRates/Tile/Tile.rfq.test.tsx

```diff
@@ -216,7 +218,7 @@ describe("Tile/rfq", () => {
       response$.complete()
     })
 
-    expect(screen.queryByRole("alert")!.textContent).toEqual(
+    expect(screen.getByRole("alert").textContent).toEqual(
```

Comment:
gonna crash the test, obvs, if alert not there ... but may be no doubt, so ok with get?

Reply:
yes, queryBy* is for when you're expecting no matches https://testing-library.com/docs/queries/about/#types-of-queries